datafusion/core/src/physical_optimizer/mod.rs

```diff
@@ -29,7 +29,6 @@ pub mod join_selection;
 pub mod limited_distinct_aggregation;
 pub mod optimizer;
 pub mod projection_pushdown;
-pub mod pruning;
```

Comment:
there is a 

```rust
pub use datafusion_physical_optimizer::*;
```

a few lines down which adds an alias to the `pruning` module in its new home